tests/test_visitors/test_ast/test_conditions/test_simplified_match_conditions.py

```diff
@@ -93,61 +85,63 @@
 
 
 @pytest.mark.parametrize(
-    'code',
+    ('code', 'as_binding'),
     [
-        '1',
-        'True',
-        'None',
-        '"string"',
-        'ns.CONST',
-        'State.REJECTED',
-        'x if x > 0',
+        ('1', ''),
+        ('True', ''),
+        ('None', ''),
+        ('"string"', ''),
+        ('ns.CONST', ''),
+        ('State.REJECTED', ''),
+        ('x if x > 0', ''),
+        ('2', ' as x'),
```

Comment:
This should be done via a second `@pytest.mark.parametrize` :)

- one for values
- one for `as`

```suggestion
        '2',
```